src/client/app/components/playlist/playlistForm.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Search from '../search';
+import SearchResults from '../search/searchResults';
+import SongList from './songList';
+
+const mapStateToProps = state => ({
+    auth: state.auth,
+    show: state.show,
+    playlist: state.playlist,
+    search: state.search,
+    nowPlaying: state.nowPlaying
+});
+
+const showCurrentPlaylist = (obj) => {
+    return obj ? Object.keys(obj).length > 1 : false;
+};
+
+const PlaylistForm = (props) => {
+    const { playlist, search, nowPlaying, dispatch, isEditPath } = props;
+    const { currentPlaylist } = playlist;
+    const { searchResults, currentSearch } = search;
+
+    if (showCurrentPlaylist(currentPlaylist)) {
+        return (
+            <div className="playlist-wrapper row">
+                <h1 className="col col-md-12 flex-horizontal-center">{currentPlaylist.dateSlug}</h1>
+                <Search />
+                {!!searchResults.length &&
```

Comment:
I feel you but it's not an uncommon React idiom, which I've seen even in their docs, so if it becomes a maintenance issue, we can refactor later